### PR TITLE
DDT-1079 - Modulize it

### PR DESCRIPTION
**Pull request by FinnAtAsseco**

### CODE REVIEW

**Review by nt-as — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -55,18 +37,64 @@
...
-		</dependency>
-		<dependency>
-			<groupId>org.hibernate.validator</groupId>
-			<artifactId>hibernate-validator</artifactId>
-			<scope>test</scope>
-		</dependency>
 	</dependencies>
 
+	<modules>
+		<module>shared-kernel-pagination</module>
```
Comment (pimg) [inline pom.xml:43]: pagination can go into infrastructure, no need for separate module

File: pom.xml [hunk 2]
```diff
@@ -55,18 +37,64 @@
...
+		<module>shared-kernel-persistence</module>
```
Comment (pimg) [inline pom.xml:44]: Persistence can be renamed to domain

File: pom.xml [hunk 3]
```diff
@@ -55,18 +37,64 @@
...
+		<module>shared-kernel-validation</module>
```
Comment (pimg) [inline pom.xml:45]: validation can go into infrastructure

File: pom.xml [hunk 4]
```diff
@@ -55,18 +37,64 @@
...
+		<module>shared-kernel-http-headers</module>
```
Comment (pimg) [inline pom.xml:46]: http headers can go into infrastructure

File: pom.xml [hunk 5]
```diff
@@ -55,18 +37,64 @@
...
+		<module>shared-kernel-errors-transfer-obj</module>
```
Comment (pimg) [inline pom.xml:47]: all error modules can be consolidated into 1

**Review by pimg — CHANGES_REQUESTED** [review]

**Review by pimg — APPROVED** [review]

**Pull request merged**